marlowe-playground-client/src/Simulation/BottomPanel.purs

```diff
@@ -0,0 +1,495 @@
+module Simulation.BottomPanel where
+
+import Classes (aHorizontal, accentBorderBottom, closeDrawerIcon, first, flex, flexLeft, flexTen, footerPanelBg, isActiveTab, minimizeIcon, rTable, rTable6cols, rTableCell, rTableEmptyRow, simulationBottomPanel, spanText)
+import Classes as Classes
+import Control.Alternative (map)
+import Data.Array (concatMap, length)
+import Data.Array as Array
+import Data.Either (Either(..))
+import Data.Eq (eq, (==))
+import Data.Foldable (foldMap)
+import Data.HeytingAlgebra (not, (||))
+import Data.Lens (to, (^.))
+import Data.List (List, toUnfoldable)
+import Data.List as List
+import Data.Map as Map
+import Data.Maybe (Maybe(..), isJust, isNothing)
+import Data.Tuple (Tuple(..))
+import Data.Tuple.Nested (uncurry5, (/\), type (/\))
+import Halogen.HTML (ClassName(ClassName), HTML, a, a_, b_, button, code_, div, h2, h3_, img, li, li_, ol, ol_, pre, section, span_, text, ul, ul_)
+import Halogen.HTML.Events (onClick)
+import Halogen.HTML.Properties (alt, class_, classes, enabled, src)
+import Marlowe.Parser (transactionInputList, transactionWarningList)
+import Marlowe.Semantics (AccountId(..), Assets(..), ChoiceId(..), Input(..), Payee(..), Payment(..), Slot(..), SlotInterval(..), Token(..), TransactionInput(..), TransactionWarning(..), ValueId(..), _accounts, _boundValues, _choices, maxTime)
+import Marlowe.Symbolic.Types.Response as R
+import Network.RemoteData (RemoteData(..), isLoading)
+import Prelude (Unit, bind, const, mempty, pure, show, unit, zero, ($), (<<<), (<>))
+import Text.Parsing.StringParser (runParser)
+import Types (FrontendState, HAction(AnalyseContract, ShowBottomPanel, ChangeSimulationView), SimulationBottomPanelView(MarloweErrorsView, MarloweWarningsView, StaticAnalysisView, CurrentStateView), View(Simulation), _Head, _analysisState, _contract, _editorErrors, _editorWarnings, _marloweState, _payments, _showBottomPanel, _simulationBottomPanelView, _slot, _state, _transactionError, _transactionWarnings)
+
+bottomPanel :: forall p. FrontendState -> HTML p HAction
+bottomPanel state =
+  div [ classes (simulationBottomPanel state) ]
+    [ div [ class_ flex ]
+        [ div [ class_ flexTen ]
```

Comment:
Up to you, but I personally feel it's worth creating `Bootstrap.flex_` helpers for things like this. If it's really just a tagged div, it's nice to express it more simply. YYMV.